lib/components.rb

```diff
@@ -128,7 +128,9 @@ def self.setup
         repo: "atlantis-builder",
         preimage: lambda do |component|
           FileUtils.cp "#{ENV["HOME"]}/.ssh/id_rsa.pub", component.directory
+          status = Status.read
```

Comment:
Ah - it re-reads the state of the world from the status.json file.  It's probably a good idea to do that before messing with state.  (I'm not sure it's necessary, but it's at least harmless, and would have to think about it, so it's easier to just leave it in.)